src/main/java/org/cactoos/scalar/And.java

```diff
@@ -38,12 +38,30 @@
  * {@link java.util.stream.Stream#forEach(java.util.function.Consumer)}
  * works:</p>
  *
- * <pre>
+ * {@code
  * new And(
- *    new ProcOf<>(input -> System.out.printf("\'%s\' ", input) ),
+ *    new ProcOf<>(input -> System.out.printf("\'%s\' ", input)),
  *    new IterableOf<>("Mary", "John", "William", "Napkin")
  * ).value(); // will print 'Mary' 'John' 'William' 'Napkin' to standard output
- * </pre>
+ *            // the result of this operation is always true
+ * }
+ *
+ * <p>This class could be also used for matching multiple boolean
```

Comment:
@Vatavuk could you also please add a note somewhere that And performs a short-circuit evaluation?